### PR TITLE
doc: clean up first two sections of FAQ

### PR DESCRIPTION
**Pull request by xiang90**
**1.Describe what this PR did**

Move what is Pouch to the top, since it is the most important one.

Clean up the language.

### CODE REVIEW

**Comment by xiang90** [comment]
Signed-off-by: xiang90 xiangli.cs@gmail.com

File: FAQ.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
 # Frequently Asked Questions
 
-## What does Pouch mean?
+## What is Pouch project?
 
-Pouch refers to some kinds of small bags. One kind is brood pouch which is used to protect very young life. This is a metaphor that Software Pouch has its responsibilty to take care of applications very closely. In another word, application is the keyword in Pouch's world.
-
-## What can Pouch bring you?
+Pouch is a tool providing container services for developers and operators. It helps to build a successful work flow for IT engineering teams. Pouch also acclerates DevOps innovations for enterprises.
```
Comment (allencloud) [inline FAQ.md]: s/acclerates/accelerates

**Comment by allencloud** [comment]
LGTM except a nit.
Thanks for your pr.  @xiang90

**Comment by xiang90** [comment]
@allencloud Do you want me to squash the commits? Or you can want to click the squash button on github.

**Comment by allencloud** [comment]
I will squash the commits by clicking squash button. Merging...

**Pull request merged**